src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work");
+    };
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped working");
+    };

```

Comment:
There is an unnecessary semicolon at the end of the `stopWork` method. Remove it to adhere to Java syntax conventions.